src/app/app-extras.module.ts

```diff
@@ -26,6 +27,15 @@ import {
     SkyDropdownModule,
     SkyPopoverModule,
     SkyDocsToolsModule
+  ],
+  providers: [
+    {
+      provide: SkyDocsToolsOptions,
+      useValue: {
+        gitRepoUrl: 'https://github.com/blackbaud/skyux-popovers',
+        packageName: '@skyux/popovers'
```

Comment:
It seems like this sort of information should come from the `package.json` file. Why do we need to declare it here?

Reply:
It's currently not possible to derive information from the package.json in our component services. We would have to add it to the SKY UX plugin. Possible, but not something that works today.